recipes/libxml2/all/conanfile.py

```diff
@@ -204,7 +204,8 @@ def build(self):
             autotools.make(["libxml2.la"])
 
             if self.options.include_utils:
-                autotools.make(["xmllint", "xmlcatalog", "xml2-config"])
+                ext = ".exe" if self.settings.os == "Windows" else ""
+                autotools.make(["xmllint" + ext, "xmlcatalog" + ext, "xml2-config"])
```

Comment:
Could be a function since it's duplicated [here](https://github.com/conan-io/conan-center-index/pull/2714/files#diff-1b40013712355d3fc292e86065e3f843R220)